terraform/resource_provider.go

```diff
@@ -231,4 +231,6 @@ don't satisfy the version constraints, or are otherwise incompatible.
 Terraform automatically discovers provider requirements from your
 configuration, including providers used in child modules. To see the
 requirements and constraints from each module, run "terraform providers".
+
```

Comment:
Minor nitpick: the error message above is hinting at some functionality of `terraform providers` that we recently removed. It now just prints out a flat set of constraints, not _constraints from each module_.

It's written vaguely enough that we could consider it still technically correct, but maybe dropping " from each module" from that sentence would be more honest? :man_shrugging:

Reply:
I'm glad you caught that, thank you, I've made that update.